packages/lib-user/src/components/shared/ProjectCard/ProjectCard.js

```diff
@@ -70,12 +42,42 @@ const StyledProjectDescription = styled(SpacedText)`
   }
 `
 
+function cardWidth (size) {
+  switch (size) {
+    case 'small':
+      return 157;
+    case 'medium':
+      return 189;
+    case 'large':
+      return 220;
+    case 'xlarge':
+      return 252;
+    default:
+      return 189;
+  }
+}
+
+function cardFontSize (size) {
+  switch (size) {
+    case 'small':
+      return '0.625rem';
+    case 'medium':
+      return '0.656rem';
+    case 'large':
+      return '0.688rem';
+    case 'xlarge':
+      return '0.688rem';
```

Comment:
I'll likely change this font size to a larger number once I actually use xlarge ProjectCard components in a page, but leaving it the same as 'large' for now.